gremlin-javascript/src/main/javascript/gremlin-javascript/package.json

```diff
@@ -40,15 +41,19 @@
     "url": "https://issues.apache.org/jira/browse/TINKERPOP"
   },
   "scripts": {
-    "test": "npm run unit-test && npm run integration-test",
-    "unit-test": "mocha test/unit/*",
+    "test": "npm run unit-test && npm run integration-test && npm run report-coverage",
+    "unit-test": "nyc --silent mocha test/unit/*",
     "integration-test": "npm run integration-test-graphson30 && npm run integration-test-graphbinary",
-    "integration-test-graphson30": "cross-env CLIENT_MIMETYPE='application/vnd.gremlin-v3.0+json' ./node_modules/mocha/bin/mocha test/integration -t 5000",
-    "integration-test-graphbinary": "cross-env CLIENT_MIMETYPE='application/vnd.graphbinary-v1.0' ./node_modules/mocha/bin/mocha test/integration -t 5000",
+    "integration-test-graphson30": "cross-env CLIENT_MIMETYPE='application/vnd.gremlin-v3.0+json' nyc --silent --no-clean ./node_modules/mocha/bin/mocha test/integration -t 5000",
+    "integration-test-graphbinary": "cross-env CLIENT_MIMETYPE='application/vnd.graphbinary-v1.0' nyc --silent --no-clean ./node_modules/mocha/bin/mocha test/integration -t 5000",
+    "report-coverage": "nyc report --reporter=text-lcov > coverage.lcov",
     "TODO": "# test other mime types like graphbinary stringd",
     "features": "npm run features-graphson30 && npm run features-graphbinary",
     "features-graphson30": "cross-env CLIENT_MIMETYPE='application/vnd.gremlin-v3.0+json' cucumber-js --require test/cucumber ../../../../../gremlin-test/features/",
     "features-graphbinary": "cross-env CLIENT_MIMETYPE='application/vnd.graphbinary-v1.0' cucumber-js --require test/cucumber ../../../../../gremlin-test/features/",
+    "features-docker": "npm run features-graphson30-docker && npm run features-graphbinary-docker",
```

Comment:
Aren't these the commands for running this in Docker?

Reply:
oops, these are unnecessary, will remove them

Reply:
removed in [2c56b8a](https://github.com/Bit-Quill/tinkerpop/pull/24/commits/2c56b8a3481ae238b2f0572ba0848a6c4b53dee3)